frontend/public/locales/en/demographics-oral-health-questions.json

```
@@ -15,6 +15,14 @@
     "question4": "4. Do you identify as a person with a disability?",
     "question4-note": "A person with a disability is a person who has a long-term difficulty or  condition, such as vision, hearing, mobility, flexibility, dexterity, pain, learning, developmental, memory or mental health-related impairments, that limit their daily activities inside or outside the home such as at school, work, or in the community in general.",
 
+    "button-back": "Back < ",
+    "button-continue": "Continue > "
+  },
+  "part2": {
+    "page-title": "Voluntary demographic and oral health questions part 2 of 2",
+
+    "question6": "What was your sex at bith? Sex refers to sex assigned at birth.",
```

```suggestion
    "question6": "What was your sex at birth? Sex refers to sex assigned at birth.",
```

(Same change can be made for French)

---

done